pkg/server/plugin/keymanager/gcpkms/gcpkms_test.go

```diff
@@ -116,6 +116,11 @@ func setupTest(t *testing.T) *pluginTest {
 }
 
 func TestConfigure(t *testing.T) {
+	getPublicKeyMaxAttempts := 11
```

Comment:
Is 11 as the max number of attempts specified somewhere?

Reply:
Yes, it is defined as a scoped const on pkg/server/plugin/keymanager/gcpkms/gcpkms.go:1032. The actual value of max attempts is 10, but since we have one attempt prior to that, the actual maximum attempts that it does is 11.

Reply:
maybe you can move that definition to the const section so you don't have to redefine it: https://github.com/spiffe/spire/blob/main/pkg/server/plugin/keymanager/gcpkms/gcpkms.go#L36

Reply:
5641933